common/src/configuration.rs

```diff
@@ -311,7 +311,7 @@ pub fn default_config() -> Config {
         default_subdir("mainnet/node_id.json"),
     )
     .unwrap();
-    cfg.set_default("base_node.mainnet.address", "http://localhost:18089")
+    cfg.set_default("base_node.mainnet.address", "/ip4/0.0.0.0/tcp/18089/http")
```

Comment:
127.0.0.1 ?

Reply:
Could be, but I think that will prevent it from accepting external inbound connections

Reply:
I thought these were tests for some reason. Yup, that's fine.